marklogic-data-hub/src/main/java/com/marklogic/hub/impl/ScaffoldingImpl.java

```diff
@@ -145,7 +145,8 @@ public static String getAbsolutePath(String first, String... more) {
                 + "/custom-modules/" + stepType.toLowerCase() + "/" + stepDefName + "/main.sjs" + ". \n");
             messageBuilder.append("It is recommended to run './gradlew -i mlWatch' so that as you modify the module, it will be automatically loaded into your application's modules database.\n");
         }
-        messageBuilder.append("Created step '" + stepName + "' of type '" + stepType + "' with default properties. It will need to be modified before usage.");
+        messageBuilder.append("Created step '" + stepName + "' of type '" + stepType + "' with default properties. The step has been deployed to staging and final databases. " +
```

Comment:
Do we know for sure that the step needs to be modified? I don't think that's true in all cases. Like I can see a custom step working as-is, though of course the user would need to modify the custom step module. I would remove this line of logging.